tests/unit/algos/list/delimiterMaster.js

```diff
@@ -0,0 +1,179 @@
+'use strict'; // eslint-disable-line strict
+
+const assert = require('assert');
+const DelimiterMaster =
+    require('../../../../lib/algos/list/delimiterMaster').DelimiterMaster;
```

Comment:
You could always require the index file instead, that would allow you to test from a API consumer perspective.